libraries/AP_HAL_ChibiOS/hwdef/MicoAir743v2/defaults.parm

```diff
@@ -0,0 +1,18 @@
+#BATTERY MONITOR ON
+BATT_MONITOR 4
+
+#enable second OSD type
+OSD_TYPE2 5
+
+#set default PWM type as DSHOT300
+MOT_PWM_TYPE 5
+
+#Serial Port defaults
```

Comment:
move all to the HWDEF file to save flash....only the protocols are needed, except for UART8 since your BT module needs a specific baud rate so that must be also included in the hwdef file

Reply:
Fixed!